packages/fast-components-react-base/src/listbox/listbox.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Listbox",
+    "description": "A listbox component's schema definition.",
+    "type": "object",
+    "id": "listbox",
+    "properties": {
+        "typeAheadPropertyKey": {
+            "title": "Type ahead property key",
+            "type": "string"
+        },
+        "labelledBy": {
+            "title": "Labelled by",
+            "type": "string"
+        },
+        "multiselectable": {
+            "title": "Multi-selectable",
+            "type": "boolean"
+        },
+        "selectedItems": {
+            "title":"Selected items",
+            "type":"string"
+        },
+        "defaultSelection": {
+            "title":"Default selection",
+            "type":"string"
+        }
+    },
+    "reactProperties": {
+        "children": {
+            "title": "Building blocks",
+            "type": "children",
+            "defaults": [
+                "text"
+            ]
```

Comment:
I think here we want our defaults to be `ListboxItem` correct?

Reply:
Yep